packages/gis/tests/asc/basic/Layer.lua

```diff
@@ -115,7 +115,8 @@ return {
 			file = filePath("test/biomassa-manaus.asc", "gis")
 		}
 
-		local expected = [[epsg     number [4326]
```

Comment:
@pedro-andrade-inpe 
Now .asc is returning the encoding, I don't know why this test is not working. 
See the error in Jenkins console.

Reply:
There is now one more space after each attribute name as the new attribute is one character larger than the previous largest attribute name.